pagopa/io-dev-api-server
# [SFEQS-1702] Add FCI cancelled requests

## Short description
This PR adds a new type of FCI signature requests with `CANCELLED` status.

## List of changes proposed in this pull request
- src/config.ts & src/types/config.ts -> Adds the new flag for generating cancelled signature requests;
- src/payloads/features/fci/signature-request.ts -> Adds ULID for cancelled signature requests;
- src/populate-persistence.ts -> Adds a function to generate cancelled signature requests;
- src/routers/features/fci/index.ts -> Adds a new condition for injecting cancelled signature requests into the message list;
-  src/utils/variables.ts -> Adds cancelled signature requests CTA. 

## How to test
Run the app with the following configuration and check with Flipper (or any other tool) if the returned message has the `CANCELLED` status: 
```
...
"fci": { "canceledCount": 1, "response": { "getFciResponseCode": 200} }
...
```
